### ISSUE TITLE
Empty sequence and qstring provided for read id xyz

### ISSUE THREAD

**Issue by avilella** [nanoporetech/dorado#542]
MinION run of 10X Genomics 5GEX cDNA, converting to pod5 with the command below, then calling with latest dorado:

```
$ pod5 convert fast5 ./fast5/*.fast5 --output output_pod5s/ --one-to-one ./fast5/

$ dorado --version
0.5.0+0d932c0

$ dorado basecaller --device cuda:0 $HOME/dorado/dna_r10.4.1_e8.2_400bps_sup@v4.1.0 output_pod5s/ > scalls.bam
[2023-12-19 16:05:53.381] [info] > Creating basecall pipeline
[2023-12-19 16:07:57.155] [info]  - set batch size for cuda:0 to 640
terminate called after throwing an instance of 'std::runtime_error'
  what():  Empty sequence and qstring provided for read id b129a2be-0495-4c98-bb00-dae1d23c76ed
Aborted (core dumped)
```

Equivalent with `--no-trim` did work:

```
$ dorado basecaller --no-trim --device cuda:0 $HOME/dorado/dna_r10.4.1_e8.2_400bps_sup@v4.1.0 output_pod5s/ > scalls.bam    
[2023-12-19 10:17:48.004] [info] > Creating basecall pipeline
[2023-12-19 10:21:02.850] [info]  - set batch size for cuda:0 to 640
[2023-12-20 08:54:52.835] [info] > Simplex reads basecalled: 11111841
[2023-12-20 08:54:52.844] [info] > Simplex reads filtered: 4379
[2023-12-20 08:54:52.844] [info] > Basecalled @ Samples/s: 3.170317e+06
[2023-12-20 08:54:54.061] [info] > Finished
```

**Comment by tijyojwad** [comment]
Hi @avilella - thanks for reporting! we've found a few instances of this and are working on a fix. We hope to have a patch release out ASAP

**Comment by tijyojwad** [comment]
Hi @avilella - this should now be fixed in v0.5.1 - https://github.com/nanoporetech/dorado#installation

**Comment by stevebaeyen** [comment]
Hi  @tijyojwad i had the same today with Dorado basecaller v0.8.0+acec121 on WGS data from Minion using hac v5.0 .
Any workarounds before a patch is released ?

**Comment by tengchn** [comment]
Hi @stevebaeyen, I have the same issue with you. I have downgraded Dorado to v0.7.3, and it works well now.

**Comment by stevebaeyen** [comment]
Hi @tengchn , i followed the advice from @tijyojwad and used the --no-trim parameter and then demuxed the samples separately. No need to downgrade (it also seems to run significantly faster than the previous sup v5 version)

**Comment by VBHerrenC** [comment]
Just had the same thing happen while trying to run Dorado v0.8.0 with SQK-RBK114-24 MinION data, was solved by using --no-trim.

**Comment by stevebaeyen** [comment]
@VBHerrenC  this issue has been solved in dorado 0.8.1
